HEN_HOUSE/egs++/geometry/egs_lattice/egs_lattice.h

```diff
@@ -0,0 +1,898 @@
+/*
+###############################################################################
+#
+#  EGSnrc egs++ lattice geometry
+#  Copyright (C) 2019 Rowan Thomson and Martin Martinov
+#
+#  This file is part of EGSnrc.
+#
+#  EGSnrc is free software: you can redistribute it and/or modify it under
+#  the terms of the GNU Affero General Public License as published by the Free
+#  Software Foundation, either version 3 of the License, or (at your option)
+#  any later version.
+#
+#  EGSnrc is distributed in the hope that it will be useful, but WITHOUT
+#  ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or 
+#  FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License
+#  for more details.
+#
+#  To see the GNU Affero General Public License at:
+# <http://www.gnu.org/licenses/>.
+#
+################################################################################
+#
+#  When egs_lattice is used for publications, please cite the following paper:
+#  Manuscript submission underway, to be replaced with title after publication
+#
+###############################################################################
+#
+#  Author:          Martin Martinov, 2019
+#
+#  Contributors:
+#
+###############################################################################
+*/
+
+
+/*! \file egs_lattice.cpp
+ *  \brief A Bravais, cubic, and hexagonal lattice geometry
+
```

Comment:
We should have `\ingroup Geometry` here. Also a general description of the geometry, and each of the inputs.

Reply:
Also `\ingroup CompositeG`